internal/core/application/feeder/service.go

```diff
@@ -0,0 +1,175 @@
+package feeder
+
+import (
+	"context"
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/tdex-network/tdex-daemon/internal/core/domain"
+
+	"github.com/tdex-network/tdex-daemon/internal/core/ports"
+)
+
+type Service struct {
+	repoManager ports.RepoManager
+	feederSvc   ports.PriceFeeder
+}
+
+func NewService(
+	feederSvc ports.PriceFeeder,
+	repoManager ports.RepoManager,
+) (Service, error) {
+	svc := Service{
+		repoManager: repoManager,
+		feederSvc:   feederSvc,
+	}
+
+	if err := svc.startPriceFeed(feederSvc); err != nil {
+		return Service{}, err
+	}
+
+	return svc, nil
+}
+
+func (s *Service) Stop(ctx context.Context) {
+	s.feederSvc.Stop(ctx)
+}
```

Comment:
This shouldn't be needed actually since the same service is already stopped at interface layer. Can we remove this method?

Reply:
I dont get u, can u elaborate on this? @altafan